src/main.rs

```diff
@@ -163,6 +165,9 @@ enum Command {
     /// Compiles the smart contract
     #[structopt(name = "build")]
     Build {
+        /// Path to the Cargo.toml of the contract to build.
```

Comment:
Nitpick: You don't end the other comments with `.` and since these will show up on the cli interface I think they should be consistent.

```suggestion
        /// Path to the Cargo.toml of the contract to build
```